docs/concepts/05_credentials/05_verification.md

```diff
@@ -3,68 +3,97 @@ id: verification
+The Verifier trusts this third party either because they trust their reputation directly or they trust a delegation structure that this Attester is part of.
+For example, a State department issuing driving licenses.
 
 For the verification process:
 
-- The Claimer needs their credential and the private key associated with their identifier
-- The Verifier needs the identifier of the trusted Attester
+-   The Claimer needs their credential and the private key associated with their identifier
+-   The Verifier needs the identifier of the trusted Attester
 
 During the verification process the Claimer wants to prove three things to the Verifier:
 
-- The credential is valid (i.e., not revoked by its Attester)
-- The attributes in the credential actually refer to the Claimer 
-- The credential contains information that is relevant for the Verifier in this use case
+-   The credential is valid (i.e., not revoked by its Attester)
+-   The attributes in the credential actually refer to the Claimer
+-   The credential contains information relevant for the Verifier for this use case
 
-## Requesting a Credential from a Claimer
+## Requesting a credential from a Claimer
 
 The Verifier may request a credential from a Claimer, providing the following data:
-- **CTypes**: which CTypes the Verifier can work with for the use case. They can provide multiple options, which have to be regarded as alternatives.
-- **TrustedAttesters**: which Attesters are considered trusted for each specified CType.
-- **RequiredProperties**: which properties for each specified CType must at least be revealed for the presentation to be considered sufficient. More on "selective disclosure" later.
-- **Challenge**: a nonce, which the Verifier can make use of to ensure that the presentation generated by the Claimer is fresh and is not being replayed by some other older interactions. More on that later.
 
-## Presenting a Credential with Selective Disclosure
+-   `cTypeHash`: Which CType hashes the Verifier can work with for the use case. They can provide multiple options, to regard as alternatives.
+-   `trustedAttesters`: Which Attesters to consider trusted for each specified CType.
+-   `requiredProperties`: Which properties for each specified CType must at least be revealed for the Verifier to consider the presentation sufficient.
 
-Given the `RequiredProperties` specified by the Verifier, the Claimer can decide how much of the information they wish to reveal before they generate the presentation and send it to the Verifier.
-If supported by the Verifier, they can choose to hide attributes and thus only disclose a subset of the original claim data.
+    :::info
+
+    [Read more on selective disclosure](#presenting-a-credential-with-selective-disclosure).
+
+    :::
+<!-- TODO: Only example I can find that shows this has it outside of the CTypes -->
```

Comment:
Is this the right place for `challenge`?

Reply:
Yes, this is the right place for the `challenge`.